Author: Andreas Beckmann <anbe@debian.org>
Description: disable cc_version_check
 checks the running kernel, not the one we are compiling for

--- a/Makefile
+++ b/Makefile
@@ -292,6 +292,7 @@ BUILD_SANITY_CHECKS = \
 	xen_sanity_check
 
 SKIP_SANITY_CHECKS	 =
+SKIP_SANITY_CHECKS	+= cc_version_check
 
 build-sanity-checks: conftest-verbose
 	@for SANITY_CHECK in $(filter-out $(SKIP_SANITY_CHECKS),$(BUILD_SANITY_CHECKS)); do \
